eva/plan_nodes/faiss_index_scan_plan.py

```diff
@@ -0,0 +1,47 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from eva.expression.function_expression import FunctionExpression
+from eva.plan_nodes.abstract_plan import AbstractPlan
+from eva.plan_nodes.types import PlanOprType
+
+
+class FaissIndexScanPlan(AbstractPlan):
+    def __init__(self, index_name: str, query_num: int, query_expr: FunctionExpression):
```

Comment:
Please add the doc string to the class. Also, consider renaming `query_name` and `query_expr`.  They aren't intuitive to me. Thanks!